packages/scss/src/commons/base.scss

```diff
@@ -96,7 +96,7 @@
 
 		@each $palette in list.join(config.$palettes, config.$states) {
 			.palette-#{$palette} {
-				@each $shade in map.keys(config.$primary) {
+				@each $shade in map.keys(config.$product) {
```

Comment:
We could introduce a $shades object here instead of relying on one of the palettes.

Reply:
And create an object just for this purpose?

Reply:
Yes if I understood the code we're juste using $primary or $product to loop over the existing shades and using those for each palette?